data_override/data_override.F90

```diff
@@ -97,7 +97,7 @@ module data_override_mod
    character(len=3)                 :: gridname
    character(len=128)               :: fieldname
    integer                          :: t_index                 !< index for time interp
-   type(horiz_interp_type), pointer :: horz_interp(:) =>NULL() !< index for horizontal spatial interp
```

Comment:
Just curious, is this also needed to bypass PGI issues? I never got the difference between pointer and allocatable, this is being allocated for sure on line 923. Why was it a pointer? Thanks heaven we don't have to consider _ALLOCATABLE :)

Reply:
@nikizadehgfdl - at one time, the fortran standard did not allow allocatable elements within a derived data type.  That is why we have so many pointers being used as allocatable elements.  Now that the standard supports it, we can change them to properly be allocatable.